nuget/runners/DotnetToolSettings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<DotNetCliTool>
+  <Commands>
+    <Command Name="nunit3-console" EntryPoint="nunit3-console.dll" Runner="dotnet" />
```

Comment:
Thanks for making this change @ejball! I'd happy enough that the Core Tool shouldn't be an issue here - more other problems. Would you just be able to change the tool name please, as we can merge this in?

Thanks for the contribution!

```suggestion
    <Command Name="nunit3" EntryPoint="nunit3-console.dll" Runner="dotnet" />
```